data/rupeshexe.json

```
@@ -1,21 +1,26 @@
 {
-  "name": "RUPESH DUBEY",
-  "displayStatsPublic": true,
+  "name": "Rupesh Dubey",
   "type": "personal",
-  "bio": "Student",
+  "displayStatsPublic": true,
+  "bio": "Open Source | **Student** | Currently Learning **Machine Learning** & **Data Science**",
   "avatar": "https://github.com/rupeshexe.png",
-  "tags": ["Python", "Java", "DSA", "Data Science"],
+  "tags": [
+    "Open Source",
+    "Java",
+    "Python",
+    "DSA",
+    "Machine Learning",
+    "Data Science"
+  ],
   "socials": [
     { "icon": "FaTwitter", "url": "https://twitter.com/rupeshtwt" },
     { "icon": "FaGithub", "url": "https://github.com/rupeshexe" },
-    {
-      "icon": "FaLinkedin",
-      "url": "https://linkedin.com/in/rupeshexe"
-    }
+    { "icon": "FaYoutube", "url": "https://youtube.com/@rupeshexe" }
```

Please add the missing comma `,` here to validate your json.  

```suggestion
    { "icon": "FaYoutube", "url": "https://youtube.com/@rupeshexe" },
```